pkg/api/fuzzer/fuzzer.go

```diff
@@ -439,5 +442,24 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 			c.FuzzNoCustom(s)
 			s.Allocatable = s.Capacity
 		},
+		func(e *api.Event, c fuzz.Continue) {
+			c.FuzzNoCustom(e)
+			e.EventTime = metav1.MicroTime{Time: time.Unix(1, 1000)}
+			if e.Count != 0 {
+				e.Series = &api.EventSeries{}
+			}
+			if e.Series != nil {
+				e.Series.HeartbeatTime = metav1.MicroTime{Time: time.Unix(2, 2000)}
+				e.Series.LastObservedTime = metav1.MicroTime{Time: time.Unix(3, 3000)}
+				e.Series.Count = e.Count
+			}
+			e.Source.Host = ""
```

Comment:
It's worse - we fuzz only intersections, but that's the cost of having internal->external->internal test...